src/analytics/cli-analytics.ts

```diff
@@ -0,0 +1,37 @@
+import axios, { Method, AxiosResponse, AxiosRequestConfig }  from 'axios';
+import * as config from './config.json';
+
+class CliAnalytics {
+
+    id: string
+    url: string
+
+    constructor (options) {
+        this.id = options.id
+        this.url = options.url
+    }
+
+    async recordCommand(command: string, metadata?: Array<any>) { 
```

Comment:
I'd stick to the word `event` instead of `command`. I'd also change `command` parameter to `eventType`